README.md

```diff
@@ -1,16 +1,52 @@
 <img src="https://cloud.githubusercontent.com/assets/668093/12567089/0ac42774-c372-11e5-97eb-00baf0fccc37.jpg" alt="OpenMRS"/>
 
-# openmrs-addOnManager
+# openmrs Add-on Manager
 
 This repository contains the openmrs-addOnManager OpenMRS Open Web App.
 
-> Add a description of what your app does here.
+> The openmrs-addOnManager is a tool used for uploading, listing, and deleting existing OWAs and modules.
 
 For further documentation about OpenMRS Open Web Apps see
 [the wiki page](https://wiki.openmrs.org/display/docs/Open+Web+Apps+Module).
 
 ## Development
 
+### Local Setup Instructions
+
+```
+# Get the project
+git clone https://github.com/openmrs/openmrs-owa-addonmanager.git
+
+# Move into the project directory
+cd into openmrs-owa-addonmanager
+
+# Install the dependencies
+npm install
+
+# Copy the webpack configuration from webpack.config.sample.js to webpack.config.js
+cp webpack.sample.config.js webpack.config.js
+
+# Locate the 'appdata/owa' directory and type the following command to get the path to the 'appdata/owa' directory and copy it to the clipboard.
+pwd | pbcopy
+
+An example of the path is: /Users/name/downloads/referenceapplication-standalone-2.6.0/appdata/owa
+
+Modify the path to look like: /Users/name/downloads/referenceapplication-standalone-2.6.0/appdata\\owa/
+
+Copy the path.
+
+# Open the webpack.config.js file, locate the getConfig function and update the config object with the following
+{
+  "LOCAL_OWA_FOLDER": "PASTE_THE_PATH_YOU_COPIED_HERE",
+  "APP_ENTRY_POINT": "http://localhost:8081/openmrs-standalone/owa/cohortbuilder/index.html"
```

Comment:
The app entry point should be relating to the add-on manager.

Reply:
yes sure.

Reply:
Thanks :)